WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -629,6 +629,21 @@ + (NSArray *)routes
     isSuccessful = [element fb_selectNextOptionWithOffset:offset error:&error];
   } else if ([order isEqualToString:@"previous"]) {
     isSuccessful = [element fb_selectPreviousOptionWithOffset:offset error:&error];
+  } else if (request.arguments[@"value"]) {
+    NSString *value = request.arguments[@"value"];
+    NSString* endValue = element.wdValue;
+    if(element.wdValue == value) {
```

Comment:
Please separate `if` with a space

Reply:
you cannot use == to compare two strings in ObjC. Consider using isEqualtoSrting: instead

Reply:
ohh sorry by mistake